packages/cli/test/commands/pull.test.ts

```diff
@@ -28,6 +28,34 @@ describe('pull', () => {
     expect(devFileHasDevEnv).toBeTruthy();
   });
 
+  it('should fail with message to pull without a link and without --env', async () => {
```

Comment:
This one doesn't seem to be passing for me locally (note that the prompt is displayed):

```
$ jest test/commands/pull.test.ts  
? Set up “/var/folders/qx/jr4274cx1_dbhytl3_rb6frc0000gn/T/tmp-47194jgRxSgyuVwVv/0/vercel-pull-next”? [Y/n] 
 FAIL  test/commands/pull.test.ts (5.929 s) 
  ● Console

    console.warn
      [Vercel API Mock] `GET /v8/projects/vercel-pull-next` was not handled.

      at test/test/mocks/client.ts:43:15
      at Layer.handle [as handle_request] (../../node_modules/express/lib/router/layer.js:95:5)
      at trim_prefix (../../node_modules/express/lib/router/index.js:317:13)
      at ../../node_modules/express/lib/router/index.js:284:7
      at Function.process_params (../../node_modules/express/lib/router/index.js:335:12)
      at next (../../node_modules/express/lib/router/index.js:275:10)
      at ../../node_modules/express/lib/router/index.js:635:15
      at next (../../node_modules/express/lib/router/index.js:260:14)

    console.warn
      [Vercel API Mock] `GET /v8/projects/vercel-pull-next` was not handled.

      at test/test/mocks/client.ts:43:15
      at Layer.handle [as handle_request] (../../node_modules/express/lib/router/layer.js:95:5)
      at trim_prefix (../../node_modules/express/lib/router/index.js:317:13)
      at ../../node_modules/express/lib/router/index.js:284:7
      at Function.process_params (../../node_modules/express/lib/router/index.js:335:12)
      at next (../../node_modules/express/lib/router/index.js:275:10)
      at ../../node_modules/express/lib/router/index.js:635:15
      at next (../../node_modules/express/lib/router/index.js:260:14)

  ● pull › should fail with message to pull without a link and without --env

    thrown: "Exceeded timeout of 5000 ms for a test.
    Use jest.setTimeout(newTimeout) to increase the timeout value, if this is a long-running test."

      at test/test/commands/pull.test.ts:31:6
      at Object.<anonymous> (test/test/commands/pull.test.ts:10:1)
      at TestScheduler.scheduleTests (../../node_modules/@jest/core/build/TestScheduler.js:317:13)
      at runJest (../../node_modules/@jest/core/build/runJest.js:407:19)
      at _run10000 (../../node_modules/@jest/core/build/cli/index.js:338:7)
      at runCLI (../../node_modules/@jest/core/build/cli/index.js:190:3)
  
Test Suites: 1 failed, 1 total
Tests:       1 failed, 5 skipped, 6 total
Snapshots:   0 total
Time:        6.047 s
Ran all test suites matching /test\/commands\/pull.test.ts/i.
```

Reply:
Huh, this was passing. I can't tell if a merge to main caused something to shift, but I did push up a change that fixes the problem.

Reply:
I guess it was because I'm running the tests from my terminal so `isTTY` was indeed true. Mocking it here seems like the correct approach.